scripts/create-archive

```diff
@@ -18,11 +20,24 @@ SOURCE_DATE_EPOCH=$(git log -1 --format=%ct)
 # - excluding .git, pkgs folder and prevent probable not so
 #   clever implementation of 'tar' which would result in
 #   an infinity loop due to tar '.'
-tar --sort=name \
-    --mtime="@${SOURCE_DATE_EPOCH}" \
-    --owner=0 --group=0 --numeric-owner \
-    --xform="s%^\./%${GIT_ARCHIVE_PREFIX}%" \
-    --exclude pkgs --exclude .git* --exclude "${GIT_TARBALL_NAME}" \
-    -cf "${GIT_TARBALL_NAME}" .
+
+if [ "$(printf '%s\n' "$TAR_VERSION" "1.28" | sort -V | head -n1)" == "1.28" ]; then
+    tar --sort=name \
+        --mtime="@${SOURCE_DATE_EPOCH}" \
+        --owner=0 --group=0 --numeric-owner \
+        --xform="s%^\./%${GIT_ARCHIVE_PREFIX}%" \
+        --exclude pkgs --exclude .git* --exclude "${GIT_TARBALL_NAME}" \
+        -cf "${GIT_TARBALL_NAME}" .
+else
+    find . -not -path './.git*' -not -path './pkgs*' -print0 | LC_ALL=C sort -z |
+    tar --no-recursion --null -T - \
+        --mtime="@${SOURCE_DATE_EPOCH}" \
+        --owner=0 --group=0 --numeric-owner \
+        --xform="s%^\./%${GIT_ARCHIVE_PREFIX}%" \
+        --exclude "${GIT_TARBALL_NAME}" \
+        -cf "${GIT_TARBALL_NAME}"
+fi
+
+gzip -n "${GIT_TARBALL_NAME}"
```

Comment:
Add -f, otherwise it interactively ask for confirmation if archive already is there (all debian packages).